pkg/pool-manager/pod_pool.go

```diff
@@ -210,7 +210,18 @@ func (p *PoolManager) initPodMap() error {
 		if err != nil {
 			continue
 		}
+
```

Comment:
commit message:

This PR **fixes an** issue.

I don't understand the second paragraph. Please explain what was the problem and how did you fix it.

Reply:
done can you take a look please